moped-editor/src/views/projects/projectView/ProjectFunding/ProjectFundingTable.js

```diff
@@ -426,6 +452,14 @@ const ProjectFundingTable = ({ handleSnackbar }) => {
         ? null
         : updateProjectFundingData.funding_description;
 
+    updateProjectFundingData.funding_source_id =
+      updateProjectFundingData.moped_fund_source?.funding_source_id;
+    delete updateProjectFundingData.moped_fund_source;
+
+    updateProjectFundingData.funding_program_id =
+      updateProjectFundingData.moped_fund_program?.funding_program_id;
+    delete updateProjectFundingData.moped_fund_program;
+
```

Comment:
the mutations are expecting the ids and not the objects, so I'm adjusting the payload here.